sql/hive/src/test/scala/org/apache/spark/sql/hive/orc/OrcQuerySuite.scala

```diff
@@ -180,7 +181,7 @@ class OrcQuerySuite extends QueryTest with BeforeAndAfterAll with OrcTest {
     // Respect `orc.compress`.
```

Comment:
Shall we change this comment too?

Reply:
Also the test name `Respect orc.compress option when compression is unset`.

Reply:
Sure. Thank you for review, @viirya .